### PR TITLE
Change units from sec -> au, au/day to day -> au, au/day

### PR DESCRIPTION
**Pull request by LuEdRaMo**
In order to solve https://github.com/PerezHz/PlanetaryEphemeris.jl/issues/19, this PR changes ephemeris input time units from seconds to days.

### CODE REVIEW

File: test/propagation.jl [hunk 1]
```diff
@@ -108,12 +108,12 @@ using LinearAlgebra: norm
 
     etv = range(sol64.t0, sol64.t[end], 5)
```
Comment (PerezHz) [inline test/propagation.jl:109]: Now that `etv` represents days rather than seconds we could rename this to `tv`

File: test/propagation.jl [hunk 2]
```diff
@@ -108,12 +108,12 @@ using LinearAlgebra: norm
...
     for et in eachindex(etv)
```
Comment (PerezHz) [inline test/propagation.jl:110]: I'm just realizing this should read (with the suggestion above):

```julia
for t in tv
```
 and corresponding changes below...

**Review by PerezHz — APPROVED** [review]
Just left a couple of comments in the tests, otherwise LGTM! Many thanks for this contribution @LuEdRaMo!

File: test/propagation.jl (hunk 1 above)
Comment (LuEdRaMo) [reply test/propagation.jl]: Done.

File: test/propagation.jl (hunk 2 above)
Comment (LuEdRaMo) [reply test/propagation.jl]: Done.

**Comment by PerezHz** [comment]
Thanks for addressing the review; I think we should bump a minor version, but otherwise just let me know when this is ready to be merged 👍

**Comment by lbenet** [comment]
Thanks @LuEdRaMo for addressing #19!

LGTM too!

**Comment by PerezHz** [comment]
Since this PR indeed fixes #19, I went ahead and updated the tests; I propose to merge as soon as we get the green lights from CI and tackle any remaining issues in future PRs.

**Comment by PerezHz** [comment]
Merged, thanks a lot @LuEdRaMo!

**Pull request merged**